app/Http/Controllers/UploadController.php

```diff
@@ -19,7 +21,19 @@ class UploadController extends BaseController
     public function upload(Request $request)
     {
         foreach ($request->allFiles() as $uploaded) {
-            $files[] = File::save($uploaded, $request->client->name, isset($request->private));
+            $files[] = retry(3, function () use ($uploaded, $request) {
+                $file = File::save(
+                    $uploaded,
+                    $request->client->name,
+                    isset($request->private)
```

Comment:
isset z false to true więc jak ktoś wyśle private: false to wciąż ustawi na prywatne

Reply:
Założenie było żeby wysyłać samo ?private, ale poprawię.